build.gradle

```diff
@@ -6,7 +6,7 @@ buildscript {
         maven { url "https://repo1.perfectomobile.com/public/repositories/maven" }
     }
     dependencies {
-        classpath 'com.android.tools.build:gradle:7.4.2'
+        classpath 'com.android.tools.build:gradle:8.5.0'
```

Comment:
This requires change in Java version in CI. Can you please update that to use Java 11? I guess that is why the project is not compiling on CI.

Reply:
this is easier said than done... Gradle doesn't want to be unpicked.  Just wasted 2 hours going in circles.

I'll come back to it some other time